db/migrate/20151015160334_change_author_and_text_to_not_null.rb

```diff
@@ -0,0 +1,6 @@
+class ChangeAuthorAndTextToNotNull < ActiveRecord::Migration
+  def change
+    change_column_null(:comments, :author, false, "")
+    change_column_null(:comments, :text, false, "")
+  end
+end
```

Comment:
Well, yes, that last parameter (empty string) of `change_column_null` replaces null values of *existing records*, so we don't need SQL statements here.

But it [still doesn't set the column's default](http://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-change_column_null) for new records, thus we do need a second migration to `change_column_default`.